Dockerfile.template.erb

```diff
@@ -77,8 +77,15 @@ ENV TINI_VERSION=0.18.0
 RUN powershell -Command "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
 
 # Fluentd depends on cool.io whose fat gem is only available for Ruby < 2.5, so need to specify --platform ruby when install Ruby > 2.5 and install msys2 to get dev tools
-RUN choco install -y ruby --version 2.6.5.1 --params "'/InstallDir:C:\ruby26'" \
-&& choco install -y msys2 --version 20210604.0.0 --params "'/NoPath /NoUpdate /InstallDir:C:\ruby26\msys64'"
+#RUN choco install -y ruby --version 2.6.5.1 --params "'/InstallDir:C:\ruby26'" \
+#&& choco install -y msys2 --version 20210604.0.0 --params "'/NoPath /NoUpdate /InstallDir:C:\ruby26\msys64'"
+
+# For avoiding stalling with docker build, we use snapshot version of msys2 until newer msys2 will be shipped.
+# NOTE: msys2 must be extracted to c:\ruby26\msys64. It should be done with -oC:\ruby26 option.
+RUN powershell -Command "Invoke-WebRequest 'https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-latest.sfx.exe' -OutFile 'msys2-base-x86_64-latest.sfx.exe'" \
+&& choco install -y ruby --version 2.6.5.1 --params "'/InstallDir:C:\ruby26'" \
+&& msys2-base-x86_64-latest.sfx.exe -y -oC:\ruby26
+
 RUN refreshenv \
 && ridk install 2 3 \
```

Comment:
How about using `ridk install 3` instead of `ridk install 2 3` and using the latest msys2 stable installer?
Is this suggestion working for you?

Reply:
I'd confirmed that w/o `ridk install 2`(update) Windows Server Core 2004 image can be built.

Reply:
`ridk install 2` instead of `ridk install 3` ?

Reply:
Fixed.